### PR TITLE
[IMP] mail: log model record and field access

### PR DESCRIPTION
**Pull request by alexkuhn**

### CODE REVIEW

File: addons/mail/static/src/model/model_manager.js [hunk 1]
```diff
@@ -997,12 +1031,25 @@ export class ModelManager {
      * @private
      * @param {Object} listener
      */
-    _markListenerToNotify(listener) {
+    _markListenerToNotify(listener, info) {
+        if (!(listener instanceof Listener)) {
+            throw new Error("Listener is not a listener", listener);
```
Comment (seb-odoo) [inline addons/mail/static/src/model/model_manager.js:1036]: 2nd param of error is reserved for something else, this should be part of the string

**Review by seb-odoo — APPROVED** [review]
@robodoo r+

**Comment by seb-odoo** [comment]
I've noticed the error/stack is actually really bad for performance. I've adapted it slightly on my PR, I guess we might want to merge it from there instead? Or reintegrate what I did here for smaller diff.

**Comment by alexkuhn** [comment]
> I've noticed the error/stack is actually really bad for performance. I've adapted it slightly on my PR, I guess we might want to merge it from there instead? Or reintegrate what I did here for smaller diff.

Makes sense to disable in debug mode then, at least.
This PR helps debugging but is lacking some info and in some aspects too noisy, which is why it hasn't been merged yet.
If you have a tweaked and improved version ready to merge, let's merge it whichever way is fastest for you.

**Comment by seb-odoo** [comment]
This is obsolete / was done differently.

**Pull request closed without merging**